src/core/mod.rs

```diff
@@ -264,7 +265,7 @@ pub struct EventStream<MSG> {
 impl<MSG> Drop for EventStream<MSG> {
     fn drop(&mut self) {
         // Ignore error since we're in a destructor.
```

Comment:
Please remove this comment as it is not useful anymore.

Reply:
Sure

Reply:
I see you marked this issue as resolved, but that comment was not removed. Did you forget to push the changes?

Reply:
Sorry, I committed it to the wrong branch. Now it is gone